pkg/cover/report_test.go

```diff
@@ -257,7 +275,49 @@ func buildTestBinary(t *testing.T, target *targets.Target, test Test, dir string
 	return bin
 }
 
-func generateReport(t *testing.T, target *targets.Target, test Test) ([]byte, []byte, error) {
+const detectGCCCode = `
+#ifdef __GNUC__
+#ifndef __clang__
+#error not clang
+#endif
+#endif
+`
+
+// Work around https://github.com/google/syzkaller/issues/4447. Do not use for anything else.
+func compilerIsGCC(t *testing.T, target *targets.Target) bool {
```

Comment:
Maybe do it like here -- by invoking the compiler with `--version`?

https://github.com/google/syzkaller/blob/1e153dc8b31e685ca8495576db4f8c077585e39c/pkg/build/build.go#L167-L175

Reply:
Done.